tests/Unit/ApparentHorizons/StrahlkorperGrTestHelpers.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/ApparentHorizons/StrahlkorperGrTestHelpers.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+tnsr::aa<DataType, SpatialDim, Frame, Index> make_spatial_ricci_schwarzschild(
+    const tnsr::A<DataType, SpatialDim, Frame, Index>& x,
```

Comment:
`x` are the spatial coordinates? I think you don't need to the `IndexType Index` template parameter since this allows x to be either spacetime or spatial and will return either a spacetime or spatial Ricci. I think we just want these tensors to be spatial right?

Reply:
Yes, these can be spatial. I'll fix this in my next revision of #529, which replaces this pull request.